v1/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "distributed-press-api",
+  "version": "0.1.0",
+  "description": "API for the Distributed Press",
+  "main": "build/index.js",
+  "type": "module",
+  "scripts": {
+    "clean": "rm -rf build",
+    "build": "npm run clean && tsc",
+    "lint": "tsc --noEmit && ts-standard --fix",
+    "dev": "ts-node-esm index.ts | pino-pretty -c -t",
+    "watch": "nodemon --watch './**/*.ts' --exec 'node --experimental-specifier-resolution=node --loader ts-node/esm' index.ts | pino-pretty -c -t",
+    "start": "node build/index.js",
```

Comment:
Would it make sense to just use `ts-node-esm` for prod as well? Or is there some advantage to doing a build first?

Reply:
Cold start for `ts-node-esm` is slower because it needs to do the compile step so if we ever containerize this would be faster